test/langtools/tools/javac/processing/ReportOnImportedModuleAnnotation/ReportOnImportedModuleAnnotation.java

```diff
@@ -49,6 +53,20 @@ public static void main(String[] args) throws Exception {
 
         final JavaCompiler compiler = ToolProvider.getSystemJavaCompiler();
 
+        // Clean any existing class files in output directory
+        Files.walkFileTree(testOutputPath,
+                           new SimpleFileVisitor<Path>() {
+                               @Override
+                               public FileVisitResult visitFile(Path path,
+                                                         BasicFileAttributes attrs) {
+                                   File file = path.toFile();
+                                   if (file.getName().endsWith(".class")) {
+                                       file.delete();
```

Comment:
This is unlikely to work reliably on Windows, where the delete can be asynchronous.  I recommend the use of the `ToolBox` test library class, and `ToolBox.delete`

Alternatively use jorge built-in guarantee that work directory is empty at the beginning of the test.